apps/src/StudioApp.js

```diff
@@ -976,7 +977,13 @@ StudioApp.prototype.onReportComplete = function (response) {
 };
 
 StudioApp.prototype.showInstructions_ = function(level, autoClose, showHints) {
+  var markdownMode = window.marked && level.markdownInstructions && this.LOCALE === ENGLISH_LOCALE;
```

Comment:
I might call this `isMarkdownMode`. I was initially expecting this variable to be an enum type variable (describing the mode) rather than a boolean.